-- Checking for module 'hyprland-protocols>=0.4.0'
--   Package 'hyprland-protocols' not found
CMake Error at /usr/local/share/cmake/Modules/FindPkgConfig.cmake:645 (message):
  The following required packages were not found:

   - hyprland-protocols>=0.4.0

--- CMakeLists.txt.orig	2024-10-08 19:23:45 UTC
+++ CMakeLists.txt
@@ -98,8 +98,12 @@ protocolwayland()
 
 protocolwayland()
 
-pkg_check_modules(hyprland_protocols_dep REQUIRED IMPORTED_TARGET hyprland-protocols>=0.4.0)
+pkg_check_modules(hyprland_protocols_dep IMPORTED_TARGET hyprland-protocols>=0.4.0)
+if(hyprland_protocols_dep_FOUND)
 pkg_get_variable(HYPRLAND_PROTOCOLS hyprland-protocols pkgdatadir)
+else()
+set(HYPRLAND_PROTOCOLS "${CMAKE_SOURCE_DIR}/subprojects/hyprland-protocols")
+endif()
 message(STATUS "hyprland-protocols dependency set to ${HYPRLAND_PROTOCOLS}")
 
 protocolnew("${HYPRLAND_PROTOCOLS}/protocols" "hyprland-ctm-control-v1" true)
